classes/Pods.php

```diff
@@ -1774,7 +1786,7 @@ public function field( $name, $single = null, $raw = false ) {
 												$size = substr( $full_field, 5 );
 											}
 
-											$value[] = pods_image( $item_id, $size );
+											$value[] = pods_image( $item_id, $size, 0, array(), true );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 57b0967bf4715905abae7e6b48bf494e</sub>